kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobActionDropdown.js

```diff
@@ -0,0 +1,202 @@
+define(['bluebird', 'common/events', 'common/html'], (Promise, Events, html) => {
+    'use strict';
+
+    const t = html.tag,
+        span = t('span'),
+        button = t('button'),
+        ul = t('ul'),
+        li = t('li'),
+        div = t('div'),
+        cssBaseClass = 'kb-job-action';
+
+    const actionArr = [
+        {
+            label: 'Cancel queued jobs',
+            action: 'cancel',
+            target: ['created', 'estimating', 'queued'],
```

Comment:
The `target` for the cancel or retry action is now an array of statuses. If we have to implement a 'Cancel all jobs' or 'Retry all jobs' dropdown option, it will now be much simpler. Hurrah!